docs/gathered-data.md

```diff
@@ -159,37 +159,39 @@ Location in archive: config/configmaps/
 See: docs/insights-archive-sample/config/configmaps
 
 
+## ContainerImages
```

Comment:
same here

Reply:
It's a documentation update from #133, I believe. Because I added `GatherContainerImages` in that PR, but I didn't know I should run some command to update the docs.

Reply:
separate doc updates and code changes to two commits at least... then i won't care much if they go in single PR.